client/src/elm/Translate.elm

```diff
@@ -17548,6 +17569,23 @@ translationSet trans =
                     , kirundi = Just "Ugukoroka kw'imbanyi (imbere y'indwi 24 ugereranije nigihe imbanyi imaze)"
                     }
 
+        PrePregnancyClassification classification ->
+            case classification of
+                PrePregnancyUnderWeight ->
+                    translationSet Underweight
+
+                PrePregnancyNormal ->
+                    translationSet Normal
+
+                PrePregnancyOverweight ->
+                    translationSet Overweight
+
+                PrePregnancyObesity ->
+                    { english = "Obesity"
+                    , kinyarwanda = Nothing
+                    , kirundi = Nothing
```

Comment:
As I see more translations as usual in this PR, I am wondering if we need all translations for this piece of feature or not.

Reply:
Right, that's why I never see the whole process :smile: